docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -571,11 +571,11 @@ Note: Builtin variables are defined in [Builtin variables](#builtin-variables) b
 #### Builtin variables
 
 It’s also possible to use so-called builtin variables in addition to user-defined variables. The following builtin variables are available:
-- `cluster.{name,namespace,labels,annotations}`
+- `cluster.{name,namespace}`
 - `cluster.topology.{version,class}`
-- `cluster.topology.controlPlane.{labels,annotations,replicas,version}`
+- `cluster.topology.controlPlane.{replicas,version}`
```

Comment:
Am I wrong or the variable names in the current PR has been simplified, e.g.
```suggestion
- `controlPlane.{replicas,version}`
```
Same below

Reply:
Yup. I didn't make those changes as they are already in the other PR. But I'll sync that.

Reply:
Done
This PR now contains the latest version including changes from the patch engine PR.